client/scraper/youtube_metadata.py

```diff
@@ -10,10 +10,13 @@
 class YoutubeMetadata(Metadata):
     prop_map = {}
 
+    def get_json_response(self, url):
```

Comment:
Generally you shouldn't edit any other methods apart from the unit tests in your unit test PR